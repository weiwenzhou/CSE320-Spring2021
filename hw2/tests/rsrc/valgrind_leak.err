==223549== Memcheck, a memory error detector
==223549== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==223549== Using Valgrind-3.15.0 and LibVEX; rerun with -h for copyright info
==223549== Command: bin/notation rsrc/algebric.ntn
==223549== 
@(#)notation.c	3.9 (C) Henry Thomas.   Release 3     Dated 12/10/91
==223549== 
==223549== HEAP SUMMARY:
==223549==     in use at exit: 16,458 bytes in 3 blocks
==223549==   total heap usage: 92 allocs, 89 frees, 34,130 bytes allocated
==223549== 
==223549== LEAK SUMMARY:
==223549==    definitely lost: 0 bytes in 0 blocks
==223549==    indirectly lost: 0 bytes in 0 blocks
==223549==      possibly lost: 0 bytes in 0 blocks
==223549==    still reachable: 16,458 bytes in 3 blocks
==223549==         suppressed: 0 bytes in 0 blocks
==223549== Reachable blocks (those to which a pointer was found) are not shown.
==223549== To see them, rerun with: --leak-check=full --show-leak-kinds=all
==223549== 
==223549== For lists of detected and suppressed errors, rerun with: -s
==223549== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
