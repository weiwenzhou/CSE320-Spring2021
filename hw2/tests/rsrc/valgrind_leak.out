
  Boudy vs Szabo, Varna 1979
    See Informant XXVII game 452

  1.       e4       c5
  2.      Nf3      Nc6
  3.       d4     cxd4
  4.     Nxd4      Nf6
  5.      Nc3       d6
  6.       f4       a6
  7.     Nxc6     bxc6
  8.       e5      Nd7
  9.     exd6     exd6
 10.      Qf3       d5
 11.      Bd2      Bd6
 12.    O-O-O      O-O
 13.       g4      Bb7  [
 13.      ...      Rb8 !?  ~   ]
 14.      Rg1      Re8
 15.       g5       f5 !?   [
 15.      ...      Nc5  ]
 16.       g6       h6
 17.      Bd3 ?!   [
 17.      Bh3 !  +_   ]
 17.      ...      Rf8
 18.      Qh3      Qf6
 19.      Rg5      Nc5
 20.     Bxf5      Bc8 ! 
 21.     Bxc8    Raxc8
 22.      Rg4      Rb8 =~ 
 23.      Rf1      Ne4
 24.      Be1

| |r| |/| |r|k|/|
|/| |/| |/| |p| |
|p|/|p|b| |q|P|p|
|/| |/|p|/| |/| |
| |/| |/|n|P|R|/|
|/| |N| |/| |/|Q|
|P|P|P|/| |/| |P|
|/| |K| |B|R|/| |


 24.      ...     Rxb2
 25.     Kxb2  [
 25.     Nxe4     dxe4
 26.      Bc3      Ba3 ! 
 27.     Bxf6      Rb3 _+  _|_   ]
 25.      ...      Ba3 ! 
 26.     Kxa3  [
 26.      Ka1      Rb8 -+   ]
 26.      ...      Rb8 -+ 
 27.     Nxe4      Qe7 ??   [
 27.      ...      Qb2
 28.      Ka4     Qxa2
 29.      Qa3     Qxc2
 30.      Ka5      Rb5
 31.     Kxa6     Qxe4 /\  De8a8 -+   ]
 28.      Nd6 !   [
 28.      Ka4     Qxe4 -+   ]
 28.      ...     Qxd6
 29.      Ka4       c5
 30.      Rh4  [ ^_ 
 30.      Rg1       c4
 31.      Qc3      Qd7
 32.      Ka3      Qd6
 33.      Qd4 -+   ]
 30.      ...       c4
 31.      Qc3      Qd7
 32.      Ka3      Qe7
 33.      Qb4 !  +-      Rxb4
 34.     Bxb4     Qxh4
 35.      Re1      Qd8 [] 
 36.       c3 !        a5
 37.      Bc5 ||   White won
  somewhat luckily in the end at move 75. 

| |/| |q| |/|k|/|
|/| |/| |/| |p| |
| |/| |/| |/|P|p|
|p| |B|p|/| |/| |
| |/|p|/| |P| |/|
|K| |P| |/| |/| |
|P|/| |/| |/| |P|
|/| |/| |R| |/| |


